### PR TITLE
remove agent tab in computer, and get agent status in auto_inventory block

### PR DESCRIPTION
**Pull request by orthagh**
Status is called by ajax, so the computer page can't stuck.
I change showInfoForComputer prototype to pass Computer object needed by ``showRemoteStatus`` method

screenshot: 
image

### CODE REVIEW

**Comment by trasher** [comment]
The URL call that previously blocks the page is now called from ajax, this should no longer blocks page load.

**Comment by orthagh** [comment]
Hello @ddurieux, like @trasher said, we manage the blocking request by calling it in **ajax** (after dom ready)
So on page display (or reload), we don't stuck waiting for answer

File: ajax/remote_status.php [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+      switch($agentStatus['message']) {
+
+         case 'executing scheduled tasks':
+         case 'running':
+            $agentStatus['message'] = __('Running');
+            break;
+
+         case 'noanswer':
+            $agentStatus['message'] = "<i class='fa fa-exclamation'></i>".
+                                      __('Impossible to communicate with agent!', 'fusioninventory');
```
Comment (wawax) [inline ajax/remote_status.php]: I would suggest to change the locale to "Impossible to reach the agent" or "Cannot contact the agent"

**Comment by orthagh** [comment]
@ddurieux, i did better.

The status is still in main form, but not auto triggered, it now needs a click to send the first request, see:

- status ko:
peek 05-12-2017 11-21

- status ok:
peek 05-12-2017 11-23

**Comment by ddurieux** [comment]
Ok for me ;)

**Pull request merged**